src/core/raps/actions/swap.ts

```diff
@@ -60,17 +60,20 @@ export const estimateSwapGasLimit = async ({
   const provider = getProvider({ chainId });
 
   if (!provider || !quote) {
-    return getChainGasUnits(chainId).basic.swap;
+    const chainGasUnits = networkStore.getState().getChainGasUnits(chainId);
```

Comment:
can pull this line out to line 61 and delete the duplicate line on 72

Reply:
Can, but only wanted to grab it when/if we needed to in order to reduce calls to the network store.